src/homekit_decl.c

```diff
@@ -24,6 +24,9 @@ void identify(homekit_value_t _value) {
     printf("accessory identify\n");
 }
 
+char device_name[19] = "XXXXXX\0";
```

Comment:
The rvalue (`XXX...`) isn't necessary here, as bss section variables are initialized to zero, and the values are immediately overwritten. You can just define them (`char device_name[19];`) uninitialized and it's fine.